app/models/security/synth_combobox_option.rb

```diff
@@ -0,0 +1,22 @@
+class Security::SynthComboboxOption
+  include ActiveModel::Model
+
+  attr_accessor :symbol, :name, :logo_url, :exchange_acronym
+
+  class << self
+    def find_in_synth(query)
+      Provider::Synth.new(ENV["SYNTH_API_KEY"])
+        .search_securities(query:, dataset: "limited", country_code: Current.family.country)
+        .securities
+        .map { |attrs| new(**attrs) }
+    end
+  end
+
+  def id
+    "#{symbol} (#{exchange_acronym})" # submitted by combobox as value
```

Comment:
Could we use something more like `"#{symbol}|#{exchange_mic}"` here, and then in the controller, we don't need all the regular expressions?  Then we'd just be able to do:

```
symbol, exchange_mic = ticker.split("|")
```

Rather than:

```rb
exchange_acronym = ticker.match(/\((.*?)\)/)&.captures&.first
ticker_symbol = ticker.gsub(/\s*\(.*?\)\s*/, "")
```